vuepress/docs/docs/middleware.md

```diff
@@ -225,3 +227,23 @@ to fields. In this case the field resolver will override the type cost, however
 We may also specify a "multipliers" argument when using arguments. This will match the argument names and use numeric argument values to multiply the base value.
 In the above case this means that a query that specifies a `limit` of `10` for the `spokenLines` field will have the base field cost multipled by 10 resulting in a total cost
 of execution of `1000`
+
+## OpenTelemetry Tracing
+
+Caliban ships with support for OpenTelemtry tracing via integration with [zio-telemetry](https://github.com/zio/zio-telemetry) via the `caliban-tracing` package.
+
+In order to use tracing, first add `caliban-tracing` to your `built.sbt`.
+
+```scala
+libraryDependencies += "com.github.ghostdogpr" %% "caliban-tracing" % "2.0.2"
+```
+
+Then add it to your schema as any other wrapper:
+
+```scala
+import caliban.tracing.TracingWrapper
+import zio.telemetry.opentelemetry.Tracing
+
+val api: GraphQL[Any] = ???
+val tracedApi = api @@ TracingWrapper.traced
```

Comment:
Maybe add more details like, for what it's going to create spans? It would be good to know what spans will look like before trying it yourself.